tests/impl.cpp

```diff
@@ -2026,7 +2026,16 @@ result_t test_mm_storel_pi(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_storeu_ps(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    float *f = impl.mTestFloatPointer1;
+    float _a[4] = {1.0, 2.0, 3.0, 4.0};
```

Comment:
Why did you pick up the sequence?

Reply:
let me use `impl.mTestFloats` instead

Reply:
You just need to rewrite it as `float _a[4];`